src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -0,0 +1,131 @@
+package bio.terra.service.snapshotbuilder;
+
+import bio.terra.model.SnapshotBuilderConcept;
+import bio.terra.model.SnapshotBuilderDomainOption;
+import bio.terra.model.SnapshotBuilderFeatureValueGroup;
+import bio.terra.model.SnapshotBuilderListOption;
+import bio.terra.model.SnapshotBuilderPrepackagedConceptSets;
+import bio.terra.model.SnapshotBuilderProgramDataOption;
+import bio.terra.model.SnapshotBuilderSettings;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SnapshotBuilderService {
+
+  @Autowired
```

Comment:
FYI, this annotation is not necessary with our version of Spring.  I know some older components in TDR still have it though.

Reply:
Just curious, as I noticed that you took this out then put it back in.  Did you run into problems without it?

Reply:
It seemed like it created some test errors, but I think that may have been other things.

Reply:
Got it!  My money's on the test errors being unrelated, but this annotation also doesn't hurt anything if it stays.